devsetup/scripts/tripleo.sh

```diff
@@ -104,6 +104,9 @@ if [ ! -f \$HOME/containers-prepare-parameters.yaml ]; then
     [ "\$RH_REGISTRY_USER" ] && [ -n "\$RH_REGISTRY_PWD" ] && login_args="--enable-registry-login"
     openstack tripleo container image prepare default \
         --output-env-file \$HOME/containers-prepare-parameters.yaml \${login_args}
+    # Adoption requires Ceph 7 (Reef) as a requirement. Instead of performing a Ceph
+    # upgrade from 6 (the default) to 7, let's try to deploy 7 in greenfield
+    sed -i "s|rhceph-6-rhel9|rhceph-7-rhel9|" $HOME/containers-prepare-parameters.yaml
```

Comment:
looks ok, and we cannot do this another way as the containers-prepare-parameters.yaml is created during the deployment just above this line

Reply:
correct, there's no other way according to my understanding of the current code.
In addition, this change must be coupled with https://review.rdoproject.org/r/c/rdo-jobs/+/55116, because in RDO we're defining `rhceph-6-tools` and I don't see (but correct me if I'm wrong) an override in `install_yamls`.

Reply:
there was a green run on the test. sanity check though for the ceph_grafana_image I don't expect it matters in this job but did we also want to update that one for consistency?

https://logserver.rdoproject.org/96/53696/12/check/periodic-adoption-multinode-to-crc-ceph/d9e2f92/undercloud/zuul/containers-prepare-parameters.yaml

```
...
      ceph_grafana_image: rhceph-6-dashboard-rhel9
...
      ceph_image: rhceph-7-rhel9
```